### ISSUE TITLE
chip's details are not shown

### ISSUE THREAD

**Issue by angel1st** [B1naryStudio/md-chips#5]
Hi there,
I have existing angularjs project and I found this extremely nice chip control. Unfortunately when I attempt to use it I struggled with two main issues:
- Appearance - please look at the attached screenshot - you can see, that label is not completely visible. This is the small issue however, comparing with the next one:

![chips](https://cloud.githubusercontent.com/assets/8547131/7833038/41a96058-046c-11e5-8b02-1dca79d99507.png)
- Missing details list - when chip is already shown in input line, if you click on it, it should open details line, from where you can choose either to close the chip or to switch between available chip items (if any). However, when I click on the chip the list does not appear. A little digging into `mdchips.js` file shows me that the problem is inside the following piece of code:
  
  scope.showMore = function(index, event){
      ....
      $timeout(function() {
          if (element[0].querySelector('#chips-list')){
      this.chipsText[this.mdTitle] = ''; 
      element[0].querySelector('#chips-list').remove();
      }
      chipsActive.appendChild(chips[0]); // -->when this line is executed, the list appears, however after that when $apply method is executed the list disappears
      });
      ....
  }

So as you can see above, the list appears and been injected into DOM, however, as long as `$apply` method executes as the part of the `digest` cycle, the list disappears.

So how can I solve both issues?

Thanks in advance

**Comment by BogdanRusinka** [comment]
Hi, could you please describe in more details how can I reproduce the first issue? According to the second issue - what type of browser do you use? Try to change mdchips.js from bower or npm with the one from github ZIP. If this will fix the problem - I'll update npm and bower again.

**Comment by angel1st** [comment]
Hi @BogdanRusinka - with regards to the second issue - the code I have just downloaded as ZIP archive in `showMore` method is identical, at leas in the mentioned part (line 193), so as expected, the list is still missing when chip is clicked. So I can confirm that the issue is not fixed by downloading the code ZIP archive.
As for the first issue - I could imagine that some of your css styles becomes overwritten by other styles in my project, thus leading to the half shown label. BTW - there was one more css issue with `.active` class since this is very common class and it is not good idea to use it as css style without any additional selector, but I fixed that already.
So in summary - the real issue for me is the second one and I suspect that it is somehow related to the fact the you are compiling `chips-active-list` html code and adding it afterwards to the DOM. As already explained, the code added successfully to DOM after line 193 is executed, but afterwards, when `$apply` executes it is removed from the DOM. The final result is missing list.

Regards

**Comment by BogdanRusinka** [comment]
Thank you for mentioning the problem, I'll be looking into it

**Comment by BogdanRusinka** [comment]
Sorry, I couldn't reproduce the issue
